### PR TITLE
Update pydantic model

### PR DESCRIPTION
**Pull request by henrikstranneheim**
## Description
*This PR adds/fixes ...*

### How to prepare for test
- [ ] ssh to hasta.scilifelab.se
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_hermes -t hermes -b update-pydantic-model`

### How to test
- [ ] Do ...

### Expected test outcome
- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

File: cg_hermes/models/workflow_deliverables.py [hunk 1]
```diff
@@ -41,21 +41,20 @@ class NfAnalysisFile(FileBase):
...
 
     format: str
-    path_index: str | None
+    path_index: str | None = None
     step: str
 
 
 class BalsamicFile(FileBase):
-    """Definition of elements in balsamic deliverables"""
+    """Definition of elements in Balsamic deliverables"""
```
Comment (diitaz93) [inline cg_hermes/models/workflow_deliverables.py:49]:
```suggestion
    """Definition of elements in Balsamic deliverables."""
```

File: tests/cli/models/test_tags.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from cg_hermes.models.tags import TagMap
+
+
+def test_instantiate_tag_map(tag_map_raw: dict):
+    """
+    Tests instantiating model."""
```
Comment (diitaz93) [inline tests/cli/models/test_tags.py:8]:
```suggestion
    """Tests instantiating model."""
```

File: tests/cli/models/test_tags.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    # WHEN instantiating an object
+    tag_map = TagMap.model_validate(tag_map_raw)
+
+    # THEN assert that it was successfully created
+    assert isinstance(tag_map, TagMap)
+
+
+def test_tag_map_tags_validator(tag_map_raw: dict):
+    """
+    Tests instantiating model."""
```
Comment (diitaz93) [inline tests/cli/models/test_tags.py:20]: any reason for this two-line docstring? resolve if intended

**Review by diitaz93 — APPROVED** [review]
Looks great 💯

**Comment by henrikstranneheim** [comment]
### Hermes export, 👍 
```

[henrik.stranneheim@hasta:~] [S_base] $ hermes export tags --workflow balsamic
2024-10-03 09:34:50 hasta.scilifelab.se cg_hermes.cli.export[17273] INFO Running export tags for workflow: balsamic
| Balsamic tags                                                  | Mandatory   | HK tags                                   | Used by               |
|----------------------------------------------------------------|-------------|-------------------------------------------|-----------------------|
| balsamic-config                                                | True        | balsamic-config                           | audit, cg             |

File: cg_hermes/models/workflow_deliverables.py (hunk 1 above)
Comment (henrikstranneheim) [reply cg_hermes/models/workflow_deliverables.py]: Fixed

File: tests/cli/models/test_tags.py (hunk 3 above)
Comment (henrikstranneheim) [reply tests/cli/models/test_tags.py]: Fixed

File: tests/cli/models/test_tags.py (hunk 2 above)
Comment (henrikstranneheim) [reply tests/cli/models/test_tags.py]: Fixed

**Comment by henrikstranneheim** [comment]
Deployed in prod.

**Pull request merged**